app/models/movie.rb

```diff
@@ -1,27 +1,34 @@
 class Movie < ActiveRecord::Base
-  belongs_to :user
-  has_many :reviews
-  has_many :users, through: :reviews
-  has_many :quotes
-  validates :title, presence: true
-  validates :synopsis, presence: true
+  belongs_to  :user
+  has_many    :reviews, dependent: :destroy
+  has_many    :users, through: :reviews
+  has_many    :quotes
+
+  validates :title,
+            :synopsis, presence: true
+
   validates :year, presence: true, inclusion: { in: 1900..Date.today.year },
     format: {
       with: /(19|20)\d{2}/i,
-      message: "should be a four-digit year"
+      message: 'should be a four-digit year'
     }
-    accepts_nested_attributes_for :reviews, reject_if: :all_blank
 
+  accepts_nested_attributes_for :reviews, reject_if: :all_blank
+
+  scope :sorted_by_rating,  -> { order(rating: :desc) }
```

Comment:
Scope is chainable and easy to handle returning objects. A nice explanation here: http://www.justinweiss.com/articles/should-you-use-scopes-or-class-methods/